airbyte-webapp/src/hooks/services/Connection/ConnectionFormService.tsx

```diff
@@ -53,11 +53,6 @@ const useConnectionForm = ({
 
   const onFormSubmit = useCallback(
     async (values: FormikConnectionFormValues, formikHelpers: FormikHelpers<FormikConnectionFormValues>) => {
-      // Set the scheduleType based on the schedule value
-      values.scheduleType = values.scheduleData?.basicSchedule
-        ? ConnectionScheduleType.basic
-        : ConnectionScheduleType.manual;
-
```

Comment:
This was accidentally re-added during a merge with the connection form refactor work.